src/main/java/dev/j3fftw/litexpansion/ItemSetup.java

```diff
@@ -87,17 +88,19 @@ private void registerMachines() {
         new Converter().register(plugin);
     }
 
-    //Disable when SlimyTreeTaps exists
+    // Register only if SlimeTech and SlimyTreeTaps do not exist
     private void registerRubber() {
-        if (!Bukkit.getServer().getPluginManager().isPluginEnabled("SlimyTreeTaps")) {
-            //Rubber
+        if (Bukkit.getServer().getPluginManager().isPluginEnabled("SlimeTech")) {
```

Comment:
Why are we prioritising SlimeTech here? SlimyTreeTaps is in 300-400 servers.

Reply:
We shouldnt priortize any the reason this check is here to add support for all rubber if there is any
we should support both imo